packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -499,7 +499,7 @@ class EditorPage {
 		await toolBarButton.click();
 	}
 
-	async navigateUp() {
+	async navigateUp( options = { recursive: false } ) {
```

Comment:
The "recursive" name makes a lot of since _inside_ the function. However, I wonder if an argument name like `toRoot` might be clearer from the perspective of calling this function. WDYT?

Reply:
Additionally, as I consider the function naming more, I wonder if `moveBlockSelectionUp` or `selectParentBlock` is clearer than `navigateUp` when considering the caller will be writing tests and does not necessarily need to understand the implementation detail of the "Navigate Up" label.

Reply:
I think renaming it to `moveBlockSelectionUp` would make more sense especially if we change the param to `toRoot`. I'll update it, thanks!